services/natural_language_understanding/v1-model-manager.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * Copyright 2020 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+module.exports = function(RED) {
+  const SERVICE_IDENTIFIER = 'natural-language-understanding',
+    NaturalLanguageUnderstandingV1 = require('ibm-watson/natural-language-understanding/v1'),
+    { IamAuthenticator } = require('ibm-watson/auth'),
+    FEATURE = 'nlu-model-mode',
+    DEFAULT_MODE = 'listModels';
+
+  var pkg = require('../../package.json'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4804371)